pnpm-workspace.yaml

```diff
@@ -1,5 +1,5 @@
 packages:
+    - 'examples/lib'
```

Comment:
why not `examples/*`

Reply:
`examples/assert_lockfile_up_to_date` has a package.json but it shouldn't be a workspace

Reply:
there's exclude for that
```
packages:
  # all packages in subdirs of packages/ and components/
  - 'packages/**'
  - 'components/**'
  # exclude packages that are inside test directories
  - '!**/test/**'
```

Reply:
yeah we should probably add that assertion too huh